### ISSUE TITLE
 chia-plot-sink doesn't work with mmx-cpu-plotter

### ISSUE THREAD

**Issue by aramod** [madMAx43v3r/mmx-binaries#34]
The parameter does not accept '-d @LOCALHOST'. Will there be support in new versions? 
It was also great to add GPU plotter supports multiple -d

**Comment by madMAx43v3r** [comment]
Windows doesn't like you using `localhost` to refer to the local host, you have to use the actual host name like `@PC-345345345`.

**Comment by aramod** [comment]
-d @PC-345345345 not work for mmx-cpu-plotter. There is no new version 2.4 for WIN10

**Comment by aramod** [comment]
Farmer Public Key (48 bytes) needs to be specified via -f, see `chia keys show`.
This is the error it shows

**Comment by madMAx43v3r** [comment]
ah yeah it only works with GPU plotter yes

**Comment by aramod** [comment]
Will copy-sink be supported in the future by mmx-cpu-plotter?

**Comment by madMAx43v3r** [comment]
yes

**Comment by aramod** [comment]
Thanks for what you do! You are a great person and will remain in history forever.

**Comment by madMAx43v3r** [comment]
Now available here: https://github.com/madMAx43v3r/chia-gigahorse

**Comment by aramod** [comment]
Dear friend, -d HOSTNAME does not work again in the new version.
The readme also does not specify the additional parameter -d, as for cuda-plotter
cpu-plotter - (-d, --finaldir arg   Final directory to copy plot in parallel (default =<tmpdir>))
cuda-plotter - ( -d, --finaldir arg   Final destinations (default = <tmpdir>, remote = @HOST))

**Comment by madMAx43v3r** [comment]
yes need to update readme, `-d @HOSTNAME` is need to make it work

**Comment by aramod** [comment]
The problem was in powershell for windows. Need extra quotes " " "@host". Everything works fine for CMD

**Comment by aramod** [comment]
After the task is ready, I get error 10093
Copy to *.plot failed with: socket() failed with: 10093
What could be the problem? Tried on different machines

**Comment by aramod** [comment]
Internet search turned up that this is related to WSAStartup

**Comment by madMAx43v3r** [comment]
oh yes, I forgot something...
